tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/deleted.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- debug:
+    msg: Start iosxr_bgp_nbr_address_family deleted integration tests connection={{ ansible_connection}}
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate_base_config.yaml
+
+- include_tasks: _populate_config.yaml
+
+- block:
+
+    - name: Delete given bgp_nbr_address_family configuration
+      become: true
+      register: result
+      cisco.iosxr.iosxr_bgp_neighbor_address_family: &deleted
+        config:
+          as_number: "65536"
+        state: deleted
+
+    - become: true
+      cisco.iosxr.iosxr_facts:
+        gather_network_resources: bgp_neighbor_address_family
```

Comment:
This is not a correct way to assert. The same code is used iosxr_facts and the module to generate facts. They will always be identical.

Reply:
Ack,made changes